### DISCUSSION TITLE
Possibility to compare the carnot predictions

### DISCUSSION THREAD

**Discussion by stauffenberg2020** [MTrab/energidataservice#622] [General]
Does/Has anyone ever tried to compare/verify the carnot predictions once the spot price for the day has been made available?

Basically to check how realistic carnot predictions are

**Comment by algopt** [comment]
Is this the equivalent for the greyed predicted price showing in Andel Energi app? If so, I would also love to have it!
Together with the Apex-Chart example, would be fantastic!
A suggestion I would also have, would be to get a table going for different Energy companies with the current formula to get to the price.
For example, Andel Energi with variable price is currently {{0.1114|float(0)}}

**Reply by MTrab** [reply]
It's all in the wiki

**Reply by algopt** [reply]
Thank you @MTrab I found the Table, Andel Energi Variable price needs update to {{0.1114|float(0)}} in here: https://github.com/MTrab/energidataservice/wiki/How-to-add-costs-of-subscription-and-delivery%3F

Also, I think this was in a previous version, 'sensor.energi_data_service_west_of_the_great_belt' does not exist in my integration, 'sensor.energi_data_service' does and is already set in the integration for me (east of the great belt)
https://github.com/MTrab/energidataservice/wiki/How-to-get-the-lowest-or-highest-price%3F

I could not find how to add the predictions to Apex Charts, I only get the already set price, can you point me in the way?

**Reply by MTrab** [reply]
https://github.com/MTrab/energidataservice/wiki/Displaying-prices-in-Apexchart-card#option-3---include-forecast-data-in-chart

**Reply by algopt** [reply]
That is using a normal custom:config-template-card I wanted the forecasted data in the custom:apexcharts-card ;)
I have a funcioning template-card working

**Reply by MTrab** [reply]
All I can say is that the shown example works (using it daily myself)
